vlib/v/gen/c/testdata/div_by_zero_is_zero.c.must_have

```diff
@@ -1 +1 @@
-int c = (int)(VSAFE_DIV_int(a,z));
+int c = (int)(VSAFE_DIV_int(a , z));
```

Comment:
Really a space both before & after?

Reply:
yes, it is needed for the other tests too